src/Processor/Gotenberg.php

```diff
@@ -92,6 +76,30 @@ public function getProcessingOptions(): array
      */
     public function getPdfFromString(string $html, array $params = [], bool $returnFilePath = false): string
     {
+        $web2printConfig = Config::getWeb2PrintConfig();
+
+        $processParams['hostUrl'] = 'http://nginx:80';
+        if (!empty($web2printConfig['gotenbergHostUrl'])) {
+            $processParams['hostUrl'] = $web2printConfig['gotenbergHostUrl'];
+        }
```

Comment:
Can be simplified, right?
```suggestion
        $processParams = [
            'hostUrl' => $web2printConfig['gotenbergHostUrl'] ?? 'http://nginx:80',
        ];
```